src/test/kotlin/io/github/orangain/prettyjsonlog/logentry/ExtractTest.kt

```diff
@@ -105,6 +105,14 @@ private val params = listOf(
                 "\tat java.base/java.util.concurrent.ThreadPoolExecutor\$Worker.run(ThreadPoolExecutor.java:642)\n" +
                 "\tat java.base/java.lang.Thread.run(Thread.java:1583)\n",
     ),
+    ExtractParam(
+        "Zap Logger Production Default",
+        """{"caller": "devorer/main.go:60", "level": "info", "msg": "application starting...", "ts": 1.723487295894624E12}""",
+        Timestamp.Parsed(Instant.parse("2024-08-12T18:28:15.894Z")),
```

Comment:
> by default the production logger uses epoch seconds which causes the time shown by the plugin to be incorrect...

I understood that zap's default timestamp is an epoch in seconds, but the plugin recognizes it as an epoch in milliseconds, which causes the timestamps to be incorrect. I will fix this issue after I merge the PR. A heuristic can be introduced to detect whether the epoch is in seconds or milliseconds.

For this reason, I would like the test code to use the actual logs as much as possible; if the zap timestamps are written in seconds, please paste them in as they are. At this time, I believe the expected timestamp will be in the 1970s, but that is not a problem. I will fix this once the heuristic is in place.

Reply:
> For this reason, I would like the test code to use the actual logs as much as possible

The actual plugin converts the value to the local time zone. However, the tests use the raw parsed value.

I accounted for this discrepancy in the updated value.